assets/airports/kaus.json

```diff
@@ -0,0 +1,4028 @@
+                ["HUB*", "BOCCK"],
+                ["IAH*", "SHYNR", "BOCCK", "IDU"],
+                ["IDU*", "WLEEE*", "BITER", "ISHOV", "BASTO"],
+                ["LFK*", "BOCCK"]
+           ]
+        },
+        "BLEWE4": {
+            "icao": "BLEWE4",
+            "name": "Blue Four",
+            "entryPoints": {
+                "ACT": ["ACT"],
+                "WINDU": ["WINDU"]
+            },
+            "body": ["BLEWE", "SEWZY", "CWK"],
+            "rwy": {
+                "KAUS17L": [ ],
+                "KAUS17R": [ ],
+                "KAUS35L": [ ],
+                "KAUS35R": [ ]
+            },
+            "draw": [
+                ["BLEWE*", "SEWZY", "CWK"],
+                ["ACT*", "BLEWE"],
+                ["WINDU*", "BLEWE"]
+           ]
+        },
+        "DXEEE1": {
+            "icao": "DXEEE1",
+            "name": "Dixie One",
+            "entryPoints": {
+                "HFMAN": ["HFMAN", "GYTON", "DOBBY", ["THELM", "A200+|S280"]],
+                "HOOLR": ["HOOLR", "NILEE", ["SKEWT", "A200+|S280"]]
+            },
+            "body": ["VIDAA", ["DXEEE", "A100+|A140-|S250"], "BAAAB", ["SMILN", "A120-"]],
+            "rwy": {
+                "KAUS17L": [["PLANX", "A80+"], "DBORD", ["TRPPN", "A60|S210"], "^SNOTT", "#355"],
+                "KAUS17R": [["PLANX", "A80+"], "DBORD", ["TRPPN", "A60|S210"], "^SNOTT", "#355"],
+                "KAUS35L": ["FAACE", ["LIPSS", "A40|S210"], "STNSN", "^TATAU", "#175"],
+                "KAUS35R": ["FAACE", ["LIPSS", "A40|S210"], "STNSN", "^TATAU", "#175"]
+            },
+            "draw": [
+                ["HFMAN*", "GYTON", "DOBBY", "THELM", "VIDAA", "PUDDY", "ORINT"],
+                ["HOOLR*", "NILEE", "SKEWT", "VIDAA"],
+                ["ORINT", "DXEEE*", "BAAAB", "SMILN", "FAACE", "LIPSS", "STNSN", "TATAU"],
+                ["SMILN", "PLANX", "DBORD", "TRPPN", "SNOTT"]
+           ]
+        },
+        "KALLA3": {
+            "icao": "KALLA3",
+            "name": "Kalla Three",
+            "entryPoints": {
+                "LLO": ["LLO"]
+            },
+            "body": ["KALLA", "CWK"],
+            "rwy": {
+                "KAUS17L": [ ],
+                "KAUS17R": [ ],
+                "KAUS35L": [ ],
+                "KAUS35R": [ ]
+            },
+            "draw": [
+                ["LLO*", "KALLA"],
+                ["KALLA*", "CWK"]
+           ]
+        },
+        "LAIKS2": {
+            "icao": "LAIKS2",
+            "name": "Lakes Two",
+            "entryPoints": {
+                "DILLO": [ ],
+                "UCOKA": ["UCOKA"]
+            },
+            "body": ["DILLO", "SZAGI", ["LAIKS", "A100+|A140-"], ["BOYZZ", "A110-"]],
+            "rwy": {
+                "KAUS17L": [["CRLOS", "A50"], ["HOUKM", "A50|S210"]],
+                "KAUS17R": [["CRLOS", "A50"], ["HOUKM", "A50|S210"]],
+                "KAUS35L": [["TRVSS", "A100-|A80+"], ["RATTT", "A60+|S220"], ["^LAIDY", "A50|S210"], "#175"],

```

Comment:
Reverse the order of the altitude restrictions, so it should be `["TRVSS", "A80+|A100-"]`